docs/concepts/macros/macro_variables.md

```diff
@@ -93,10 +95,20 @@ All predefined macro variables:
     * @end_millis
     * @execution_millis
 
-Other macro variables:
+### Runtime variables
+
+SQLMesh provides two other predefined variables used to modify model behavior based on information available at runtime.
 
 * @runtime_stage - A string value that denotes the current stage of the SQLMesh runtime. It can take one of the following values:
     * 'loading' - The project is currently being loaded into SQLMesh's runtime context.
     * 'creating' - The model tables are being created.
     * 'evaluating' - The models' logic is being evaluated.
```

Comment:
Forgot to update this in https://github.com/TobikoData/sqlmesh/commit/adcd016784e3c89d13d0cd8ec15135195106977d

```suggestion
    * 'evaluating' - The models' logic is being evaluated.
    * 'testing' - The models' logic is being evaluated in the context of a unit test.
```